flowetl/tests/unit/test_qa_check_discovery.py

```diff
@@ -121,7 +121,7 @@ def test_module_path_added_to_dag_template_locations():
     dag = DAG("DUMMY_DAG", start_date=datetime.now())
     get_qa_checks(dag=dag)
     assert (
-        str(Path(__file__).parent.parent.parent / "flowetl" / "flowetl" / "qa_checks")
+        Path(__file__).parent.parent.parent / "flowetl" / "flowetl" / "qa_checks"
```

Comment:
Do we need the extra layer of 'flowetl' in the tree, or can that be flattened out?